usbutils/portable.patch

```diff
@@ -0,0 +1,396 @@
+From dc9e574df2c9d276cd0f46e8bd1af03b1b71cf17 Mon Sep 17 00:00:00 2001
+From: Tormod Volden <debian.tormod@gmail.com>
+Date: Fri, 19 Nov 2021 18:31:57 +0100
+Subject: [PATCH 1/7] Build conditionally with libudev
+
```

Comment:
Should it be mentioned where this is copied from (the usbutils merge request)?

Reply:
Do you know correct format to mention it?

Honestly, I don't know that much about providing patchsets.

Reply:
I don't know either what is customary in homebrew, but it would make sense to do it. One could just make up a header like "Reference: <link to PR>".

Reply:
Added `X-Origin:`